mutating-webhook/build-artifact.sh

```diff
@@ -3,21 +3,21 @@
 if (( $# < 1 )); then
     echo "./build-artifact.sh <latest | versioned>"
 fi
-
+cd ../
 artifacttype=$1
 
 if [ "$artifacttype" = "latest" ]; then
-    export GO111MODULE=on; CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o crd-hook
-    docker build --no-cache -t gcr.io/cloudark-kubeplus/pac-mutating-admission-webhook:latest .
+    # export GO111MODULE=on; CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o crd-hook
```

Comment:
crd-hook binary is being built in the builder and then copied out. 
I see!

Reply:
Yeah, because I want to keep the image small, we can build it in separate a step and copy it into the final images. This way, we also make sure we build inside a clean and consistent environment.